spec/unit/kiwi_config_spec.rb

```diff
@@ -224,6 +231,21 @@
       expect(config.sh).not_to match(/systemctl \w+-runtime /)
     end
 
+    context "if an empty docker service scope is provided" do
+      it "does not raise" do
```

Comment:
No, it is not necessary anymore but I started with the test and then debugged the issue.